tests/integration/test_lambda.py

```diff
@@ -1587,6 +1590,34 @@ def test_ruby_lambda_running_in_docker(self):
         testutil.delete_lambda_function(TEST_LAMBDA_NAME_RUBY)
 
 
+class TestGolangRuntimes(LambdaTestBase):
+    @classmethod
+    def setUpClass(cls):
+        cls.lambda_client = aws_stack.connect_to_service("lambda")
+
+    def test_golang_lambda_running_locally(self):
+        if use_docker():
+            return
+
+        testutil.create_lambda_function(
+            func_name=TEST_LAMBDA_NAME_GOLANG,
+            zip_file=load_file(TEST_LAMBDA_GOLANG, mode="rb"),
+            handler="handler",
+            runtime=LAMBDA_RUNTIME_GOLANG,
+        )
+        result = self.lambda_client.invoke(
+            FunctionName=TEST_LAMBDA_NAME_GOLANG, Payload=b'{"name":"test"}'
+        )
+        result_data = result["Payload"].read()
+
+        self.assertEqual(200, result["StatusCode"])
+        print(to_str(result_data))
```

Comment:
nit: let's better remove the `print` statement

Reply:
yes, my bad. I was testing the resulting message and forgot to deleted the print.